docs/pages/platform/upgrading/2.0.mdx

```diff
@@ -23,27 +23,55 @@ Breaking changes:
 
 ## Step-wise migration guide
 
-I found the following order of mechanical changes to be most ideal path to upgrade. You can make a Git commit after every one of these changes.
+I found the following order of mechanical changes to be most ideal path to
+upgrade. You can make a Git commit after every one of these changes.
 
-1. Start with a clean slate, run `tsc` to make sure your project contains no TypeScript errors to begin with.
+1. Start with a clean slate, run `tsc` to make sure your project contains no
+   TypeScript errors to begin with.
 1. Replace `@liveblocks/react-comments` by `@liveblocks/react-ui`:
-   1. Run the codemod: `npx @liveblocks/codemod@latest react-comments-to-react-ui`
+   1. Run the codemod:
+      `npx @liveblocks/codemod@latest react-comments-to-react-ui`
    1. Run `npm uninstall @liveblocks/react-comments`
    1. Run `npm i @liveblocks/react-ui@latest`
+   1. Do a search for `@liveblocks/react-comments` to find any remaining
+      references and rename them manually, for example if you import the default
+      styles from within your own CSS files using `@import`.
```

Comment:
Is this still necessary after you fixed the codemod? Or won't the codemod change any css files?

Reply:
Yes, the codemods are JS/TS-only unfortunately 😕 But I think we could do a simpler find-and-replace in all CSS files for this use-case outside of the codemod itself, I'll merge this for now but tackle this tomorrow in a separate PR.